src/didactPanelSerializer.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+import { WebviewPanel, WebviewPanelSerializer, ExtensionContext, Uri } from 'vscode';
+import { DidactPanel } from './didactPanel';
+
+const base64 = require('base-64');
+
+export class DidactPanelSerializer implements WebviewPanelSerializer {
+	
+	private _context: ExtensionContext;
+
+	constructor(context: ExtensionContext) {
+		this._context = context;
+	}
+
+	public async deserializeWebviewPanel(webviewPanel: WebviewPanel, state: any) : Promise<void> {
```

Comment:
Not sure it's needed. There are TWO items in this state, so it's very very minimal. If it was more complex, I might agree.

Reply:
deactivate the eslint rule which is forbiding usage of any then

Reply:
@apupier I thought you were against deactivating eslint rules?

Reply:
Added the necessary rule disablement

Reply:
I prefer not deactivating them without good reasons but the rules are not followed, it is then better to deactivate them. it avoids to have noise of rule that you choose to not follow.
It is better to introduce a type instead of disabling the rule. But you do not want. So it remains better to to disable the rule if it is not followed.
By not creatign the type, the type-safe part of TypeScript language is not leveraged. We will hit all the troubles of non-typed languages.